client/src/components/profile/components/camper.css

```diff
@@ -6,3 +6,9 @@
   display: flex;
   justify-content: center;
 }
+
+.username,
+.name,
```

Comment:
```suggestion
.name,
.location,
```
`location` can also be arbitrarily long.

Reply:
@ojeytonwilliams Good catch. Pushing in a sec.